File: BZSwift/0225test/0225test/ViewController.m
//
//  ViewController.m
//  0225test
//
//  Created by ZBin on 16/2/25.
//  Copyright © 2016年 BZ. All rights reserved.
//

#import "ViewController.h"
#import "BZSegment.h"
@interface ViewController ()

@property(nonatomic,strong)UISegmentedControl *bzSegment;

@end

@implementation ViewController

-(UISegmentedControl *)bzSegment{
    if (!_bzSegment) {
        self.bzSegment = [BZSegment initSegentmentControl];
    }
    return _bzSegment;
}

- (void)viewDidLoad {
    [super viewDidLoad];

    [self.view addSubview:self.bzSegment];
    NSLog(@"sssssss%@",self.bzSegment);
//    [[[NSBundle mainBundle] loadNibNamed:@"HWEmotionPopView" owner:nil options:nil]lastObject];
//    UIWindow *window = [[UIApplication sharedApplication].windows lastObject];
//    [window addSubview:segmentedController];
    
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
    
}

@end

File: ZBSwift/GIFBackgroundDemo/GIFBackgroundDemo/ViewController.swift
//
//  ViewController.swift
//  GIFBackgroundDemo
//
//  Created by ZBin on 15/11/11.
//  Copyright © 2015年 ZBin. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var webview: UIWebView!
    override func viewDidLoad() {
        let filePath = NSBundle.mainBundle().pathForResource("2", ofType: "gif")
        let data = NSData(contentsOfFile: filePath!)
        webview.loadData(data!, MIMEType: "image/gif", textEncodingName:String(), baseURL: NSURL())
        super.viewDidLoad()
        webview.userInteractionEnabled = false
        
    }

    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

